x-pack/plugins/index_management/public/application/constants/data_streams.ts

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
[nit] while this is being used for data streams, i think it could be applicable elsewhere in index management. you might rename the file to `time_units.ts` (or something to that effect)